source/uwp/AdaptiveCards.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.26403.7
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.28803.452
```

Comment:
We can't update to VS 16 yet -- our CI hosts don't have it available yet...

Reply:
Rolled back to VS 15.